server/schedule/replica_checker.go

```diff
@@ -83,7 +83,7 @@ func (r *ReplicaChecker) Check(region *core.RegionInfo) *Operator {
 
 // SelectBestReplacedPeerToAddReplica returns a new peer that to be used to replace the old peer and distinct score.
 func (r *ReplicaChecker) SelectBestReplacedPeerToAddReplica(region *core.RegionInfo, oldPeer *metapb.Peer, filters ...Filter) *metapb.Peer {
-	storeID, _ := r.selectBestReplacementStore(region, oldPeer, filters...)
+	storeID, _ := r.SelectBestReplacementStore(region, oldPeer, filters...)
```

Comment:
I think we can remove the function `SelectBestReplacedPeerToAddReplica` to prevent we misuse it later.